id-repository/credential-service/src/main/java/io/mosip/credentialstore/util/Utilities.java

```diff
@@ -81,10 +82,19 @@ public String generateId() {
 		return UUID.randomUUID().toString();
 	}
 	
-	public String generatePin() {
-		return  RandomStringUtils.randomAlphabetic(5);
+	public String generatePin() {return  randomString(5);}
+
+	public static String randomString(int length) {
+		String source = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+		SecureRandom secureRnd = new SecureRandom();
+		StringBuilder sb = new StringBuilder(length);
+		for (int i = 0; i < length; i++) {
+			sb.append(source.charAt(secureRnd.nextInt(source.length())));
```

Comment:
we can assign the source.length() to a variable outside the loop to avoid repeated call to that.